tools/prune-old-builds.py

```diff
@@ -0,0 +1,50 @@
+import os
+import shutil
+import subprocess
+from pathlib import Path
+from datetime import datetime
+
+FOURTEEN_DAYS = 14
+
+
+def how_old(date: str) -> int:
+    """Return a integer that represents how many days has passed given a date"""
+    return (datetime.now() - datetime.strptime(date, '%Y-%m-%d')).days
+
+
+if __name__ == '__main__':
+    builds_path = Path('builds/')
+    repo_url = 'https://github.com/quick-lint/quick-lint-js.git'
+    repo_name = 'quick-lint-js'
+
+    if os.path.exists(repo_name):
+        shutil.rmtree(repo_name)
+
+    subprocess.run(['git', 'clone', '-q', repo_url])
```

Comment:
It seems wasteful that we download and delete the repo every time we run prune-old-builds. I think we should cache the download.

Reply:
With cache you mean keep the repository in the server, right?